supplier_app/tests/__init__.py

```diff
@@ -0,0 +1,56 @@
+from supplier_app import (
+    PAYMENT_TERMS,
+    PAYMENT_TYPES,
+)
+from supplier_app.bank_info import BANK_INFO
+
+
+def get_bank_info_example(key=None):
+    return BANK_INFO[key or "BANCO DE LA NACION ARGENTINA"]
```

Comment:
I'm wondering where are we using this function

Reply:
I use it "TestCreateTaxPayer" when I create a bank_info